packages/manager/apps/web/client/app/hosting/multisite/hosting-multisite.controller.js

```diff
@@ -91,6 +89,12 @@ angular
         value: period,
       }));
 
+      HostingDomain.getZones()
+        .then((zones) => {
+          $scope.listZone = zones;
+        })
+        .catch(() => []);
```

Comment:
`listZone` should be set here instead of returning a non use value
```suggestion
        .catch(() => {
           $scope.listZone = []; 
        });
```

Reply:
good catch, the monday morning ;-)